frontend/pages/queries/details/QueryDetailsPage/QueryDetailsPage.tsx

```diff
@@ -81,6 +81,7 @@ const QueryDetailsPage = ({
     lastEditedQueryObserverCanRun,
     lastEditedQueryDiscardData,
     lastEditedQueryLoggingType,
+    lastEditedQueryFrequency,
```

Comment:
What this is fixing was already fixed, please revert: https://github.com/fleetdm/fleet/pull/14523/files